### DISCUSSION TITLE
will v4 still support `useErrorBoundary` function for mutations?

### DISCUSSION THREAD

**Discussion by stefanprobst** [TanStack/query#3379] [Q&A] [answered]
hi, this might be just because v4 is still alpha, but it looks like in current `master`, i can pass a function for `useErrorBoundary` on mutations:

https://github.com/tannerlinsley/react-query/blob/7be6d89d689eb61a057d238c95b6701c2a92b7f5/src/react/useMutation.ts#L119

vs. in the alpha branch it's cast to a boolean?

https://github.com/tannerlinsley/react-query/blob/bc7263d43ad628dfe09e244676d625963ec350db/src/reactjs/useMutation.ts#L119

**Comment by TkDodo** [comment, marked as answer]
I think that's a mistake. I'll check when I'm back from vacation.

**Reply by TkDodo** [reply]
Thanks for reporting. I have a fix here:
- https://github.com/tannerlinsley/react-query/pull/3390
